python/twilio/twilio-k8s/webapp-dep.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: twilio-vision-webapp
```

Comment:
I believe you need to add the app: label to the Deployment for the delete -l command to work.